openedx/core/djangoapps/content/learning_sequences/api/outlines.py

```diff
@@ -85,7 +85,7 @@ def key_supports_outlines(opaque_key: OpaqueKey) -> bool:
     return False
 
 
-def public_api_available(course_key: CourseKey, user: types.User) -> bool:
+def public_api_available(course_key: CourseKey, user: types.User = None) -> bool:  # pylint: disable=unused-argument
```

Comment:
Why not just remove this `user` parameter altogether? Will we be shifting back to using it at some point?

Reply:
I guess I'm just averse to breaking backwards compatibility with something we export as part of the app's Python API, even if realistically nobody else is using it.

I'll get rid of the user.